CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [2.0.0](https://github.com/ASFHyP3/hyp3/compare/v1.1.6...v1.1.7)
+### Changed
+- Removed `connexion` library due to inactivity, replaced with `open-api-core`
+- Error messages from invalid input according to the `api-spec` are different
```

Comment:
The error messages note isn't specific enough to be useful. For example, what would a Vertex developer need to know to accommodate the change?